Applications/Terminal/Terminal.cpp

```diff
@@ -135,6 +134,36 @@ static inline Color lookup_color(unsigned color)
     return Color::from_rgb(xterm_colors[color]);
 }
 
+void Terminal::escape$l(const ParamVector& params)
+{
+    int mode = 2;
+    if (params.size() > 0) {
+        mode = params[0];
+    }
+    if (!m_question_param) {
+        switch (mode) {
+            // FIXME: implement *something* for this
+        default:
+            unimplemented_escape();
+            break;
+        }
+    } else {
+        switch (mode) {
+        case 25:
+            // Hide cursor command, but doesn't need to be run (for now, because
```

Comment:
Consider adding a dbgprintf at the least, so it's obvious if someone is looking to implement the functionality later on where they need to edit.